templates/output/thumbnail-card.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Card } from '@oacore/design'
+import { classNames } from '@oacore/design/lib/utils'
+import filesize from 'filesize'
+
+import styles from './thumbnail-card.module.css'
+
+const FullTextThumbnail = ({
+  data: { title, size: fileSize, type: fileType, id: documentId },
+  id,
+  src = `//core.ac.uk/image/${documentId}/large`,
+  alt = 'Document thumbnail',
+  href = `//core.ac.uk/reader/${documentId}`,
+  className,
+  ...passProps
+}) => (
+  <Card
+    id={id}
+    className={classNames.use(styles.container).join(className)}
+    {...passProps}
+  >
+    <h2 className={styles.title} id={`${id}-title`}>
+      Full text
+    </h2>
+
+    <a
+      className={styles.image}
+      href={href}
+      aria-labelledby={`${id}-title`}
+      aria-describedby={`${id}-body`}
+    >
+      <img className={styles.image} src={src} alt={alt} />
+    </a>
```

Comment:
it was confusing for me that the link includes only the image but not the full card. Is this expected?

Reply:
It's not what might be expected by the user but doing so properly is quite difficult. So, I am postponing this difficulty lowering chance of mistake by a big image.

Reply:
Sorry to be naive but It wouldn't just be moving the `<p>` inside the `<a>`?

Reply:
Syntactically and even semantically it's acceptable but logically it's not. Read about [inclusive cards](https://inclusive-components.design/cards/) if you wish to find out more.

Reply:
The points made by the link (thanks for that) are all worthy but they apply on a generic card that contains multiple links and call to actions. In here the card is actually a big button so I don't think there is a need to penalise average users for a possible accessibility issue in the future?

> This is not without its problems. Now, all of the card contents form the label of the link. So when a screen reader encounters it, the announcement might be something link "Card design woes, ten common pitfalls to avoid when designing card components, by Heydon Pickering, link".
It's not disastrous in terms of comprehension, but verbose — **especially if the card evolves to contain more content** —

I think we are not in the "especially" side here yet.

Reply:
According to the original design, the card is going to contain at least 3 links, each having its own description. I stuck with that.